froide_fax/fax.py

```diff
@@ -4,28 +4,26 @@
 
 from twilio.rest import Client
 
-from froide.foirequest.models import (
-    FoiMessage, FoiAttachment, DeliveryStatus
-)
+from froide.foirequest.models import FoiMessage, FoiAttachment, DeliveryStatus
 from froide.foirequest.message_handlers import MessageHandler
 from froide.foirequest.models.message import MessageKind
 
 from .pdf_generator import FaxMessagePDFGenerator
-from .utils import (
-    get_media_url, get_status_callback_url, ensure_fax_number
-)
+from .utils import get_media_url, get_status_callback_url, ensure_fax_number
+
+import requests
```

Comment:
I haven't added `isort`, but I try to keep an order with import sections: builtin, django, third-party, froide, relative.